tests/tests/parsers/code_12a0.log

```diff
@@ -1,9 +1,14 @@
 
 2021-02-12T20:59:47.225210 066 RP --- 20:008749 18:142609 --:------ 12A0 002 00EF            # {'indoor_humidity': None}
+2024-10-15T12:41:34.415872 075  I --- 29:099029 --:------ 29:099029 12A0 002 0031            # {'indoor_humidity': 0.49}
 
 2022-03-08T11:24:21.138647 076  I --- 29:146052 --:------ 29:146052 12A0 006 00220685004C    # {'indoor_humidity': 0.34, 'temperature': 16.69, 'dewpoint_temp': 0.76}
 
 2022-07-03T10:04:21.953850 061 RP --- 32:155617 18:005904 --:------ 12A0 007 003908A57FFF00  # {'indoor_humidity': 0.57, 'temperature': 22.13, 'dewpoint_temp': None}
 2022-08-17T02:29:36.718767 056  I --- 32:155617 --:------ 32:155617 12A0 007 003C096E7FFF00  # {'indoor_humidity': 0.60, 'temperature': 24.14, 'dewpoint_temp': None}
 
 2019-12-16T05:19:20.630177 072  I --- 32:168090 --:------ 32:168090 12A0 006 003C06DE03E3    # {'indoor_humidity': 0.60, 'temperature': 17.58, 'dewpoint_temp': 9.95}
+
+# ClimaRad VenturaV1x
+2024-10-15T12:33:21.828292 060  I --- 37:153226 --:------ 37:153226 12A0 021 003307DD7FFF0001EF7FFF7FFF00023E056B02A600  # {'indoor_humidity': 0.51, 'temperature': 20.13, 'dewpoint_temp': None, 'units': 'Celsius', 'co2_level': 574, 'supply_temp': 13.87, 'outdoor_temp': 6.78}
+2024-10-15T12:38:22.145126 060  I --- 37:153226 --:------ 37:153226 12A0 021 003207D27FFF0001EF7FFF7FFF00023F057B02D200  # {'indoor_humidity': 0.5, 'temperature': 20.02, 'dewpoint_temp': None, 'units': 'Celsius', 'co2_level': 575, 'supply_temp': 14.03, 'outdoor_temp': 7.22}
```

Comment:
Can you justify your value names, e.g.:
- `SZ_OUTDOOR_TEMP` & not `SZ_INDOOR_TEMP`
- why 'units'

Reply:
Done, added 6 including one just now:
Outside temp is currently 18.1 C reported by a NIBE sensor on the same facade,
Indoor temp 21.6 C measured by room thermostat.
Indoor humidity 62% according to room thermostat and confirmed by an analog hygrometer.
Meteo says outside hum. is 82%, Dewpoint 17 C. 

Current packet from Ventura HRU
```
2024-10-18T15:45:27.762626 067  I --- 37:153226 --:------ 37:153226 12A0 021 004408327FFF0001EF7FFF7FFF00024707EF05DE00
```
so reads is:
{'indoor_humidity': 0.68, 'temperature': 20.98, 'dewpoint_temp': None, 'units': 'Celsius', 'co2_level': 583, 'supply_temp': 20.31, 'outdoor_temp': 15.02}

5 Mins later, with fan on speed 2 manual:
```
2024-10-18T15:50:28.112562 067  I --- 37:153226 --:------ 37:153226 12A0 021 004408377FFF0001EF7FFF7FFF00024707F705E000
```

I admit that 'units' is a guess on my part. I don't know if this product would support F temperatures (docs don't tell me how to set it either, telling me "the installer will take care of everything"). It could just as well be an indicator for the summer bypass position (also rather static in October). I changed that to an "unknown" field.